examples/with-tailwindcss/src/components/Button.astro

```diff
@@ -1,3 +1,6 @@
-<button class="py-2 px-4 bg-green-500 text-white font-semibold rounded-lg shadow-md hover:bg-green-700 focus:outline-none focus:ring-2 focus:ring-green-400 focus:ring-opacity-75">
+---
+let { type = 'button' } = Astro.props;
```

Comment:
Added this prop here just to test that frontmatter doesn’t interfere with Tailwind’s scanner (it seems to not; it’s pretty forgiving because it has to work in React, etc.)